Library/Formula/kibana.rb

```diff
@@ -0,0 +1,63 @@
+class Kibana < Formula
+  desc "Visualization tool for elasticsearch"
+  homepage "https://www.elastic.co/products/kibana"
+  url "https://github.com/elastic/kibana/archive/v4.0.2.tar.gz"
+  sha256 "facb5b2eb31de38ebf6e30bb656c62a512e540623df98b33655f5b0da896dd27"
+  head "https://github.com/elastic/kibana.git"
+
+  depends_on "node"
+
+  def install
+    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
+    system "npm", "install"
+    system "npm", "install", "grunt-cli"
+    system "npm", "install", "bower"
+    system "./node_modules/.bin/bower", "install"
+    system "./node_modules/.bin/grunt", "build", "--force"
+
+    dist_dir = Pathname("build/dist/kibana")
+
+    rm_f dist_dir/"bin/*.bat"
+
+    prefix.install dist_dir/"src"
+    (etc/"kibana").mkpath
+    (var/"lib/kibana/plugins").mkpath
+
+    (etc/"kibana").install dist_dir/"config/kibana.yml" unless (etc/"kibana/kibana.yml").exist?
+
+    # point to our node
+    inreplace dist_dir/"bin/kibana" do |s|
+      s.sub! /^NODE=.*$/, "NODE=#{Formula["node"].opt_bin}/node"
+    end
+
+    bin.install dist_dir/"bin/kibana"
+  end
+
+  def post_install
+    (prefix/"config").install_symlink etc/"kibana/kibana.yml"
+    prefix.install_symlink var/"lib/kibana/plugins"
+  end
+
+  plist_options :manual => "kibana"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN"
+    "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>Program</key>
+        <string>#{opt_bin}/kibana</string>
+        <key>RunAtLoad</key>
+        <true/>
+      </dict>
+    </plist>
+  EOS
+  end
+
+  test do
+    assert_equal "4.0.2\n", `kibana -V`
```

Comment:
Make this non-dependant on it being bumped each time. `assert_equal /#{version}/, shell_output("#{bin}/kibana -V")` should work.